platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/Validator.java

```diff
@@ -100,20 +111,61 @@ public interface Validator {
      *
      * <p>This configuration can for example be used to specify errors that are expected and can be ignored.
      */
-    class LogErrorConfig {
+    interface LogFilter {
+        /**
+         * Specifies how the {@link LogMarkerFilter} interprets the marker set.
+         */
+        enum Mode {
```

Comment:
Nit: The enum will be visible. Not sure if this is intended. Does not matter as we may change the API completely anyway. 🙂